<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelRYB">
      <property name="text">
       <string>Red Yellow Blue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonRYB">
      <property name="text">
       <string>RYB</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>108</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelYRB">
      <property name="text">
       <string>Yellow Red Blue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonYRB">
      <property name="text">
       <string>YRB</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>82</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="labelHHH">
      <property name="text">
       <string>Finish</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonHHH">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>29</height>
       </size>
      </property>
      <property name="text">
       <string>HHH</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>108</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="labelRBY">
      <property name="text">
       <string>Red Blue Yellow</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonRBY">
      <property name="text">
       <string> RBY</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>108</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="labelBRY">
      <property name="text">
       <string>Blue Red Yellow</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBRY">
      <property name="text">
       <string>BRY</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>108</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="labelBYR">
      <property name="text">
       <string>Blue Yellow Red</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBYR">
      <property name="text">
       <string>BYR</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>108</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="labelYBR">
      <property name="text">
       <string>Yellow Blue Red</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonYBR">
      <property name="text">
       <string>YBR</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>80</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Индикация</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelIND">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
